test/pipelinerun_test.go

```diff
@@ -137,8 +128,18 @@ func TestPipelineRun(t *testing.T) {
 
 	for i, td := range tds {
 		t.Run(td.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I think you need to create copy of `td` variable to avoid race condition. 
Ref: https://golang.org/doc/faq#closures_and_goroutines

Reply:
it might be the reason for the test failure
```
info	TestPipelineRun/multiple_tasks	test/init_test.go:74	Deleting namespace arendelle-tywjgsyl
I1031 22:04:16.664] --- FAIL: TestPipelineRun (0.00s)
I1031 22:04:16.664]     --- FAIL: TestPipelineRun/service_account_propogation (98.27s)
I1031 22:04:16.665]     	testing.go:730: race detected during execution of test
I1031 22:04:16.665]     --- FAIL: TestPipelineRun/multiple_tasks (128.22s)
I1031 22:04:16.665]     	testing.go:730: race detected during execution of test
```
or was it a race condition trying to delete both namespaces?

Reply:
>  I think you need to create copy of td variable to avoid race condition.

Whoops, thanks @shashwathi ! Gets me every time :sweat_smile: